frontend/integration-tests/views/crud.view.ts

```diff
@@ -68,10 +69,14 @@ export const filterForName = async (name: string) => {
 };
 
 const actionOnKind = (action: string, kind: string) => {
-  return `${action} ${kind}`;
+  const humanizedKind = (kind.includes('~') ? kind.split('~')[2] : kind)
```

Comment:
are you sure kind should be split ?
"ConfigMap" is "ConfigMap" not "Config Map" maybe there is a typo and someone wrote "Config Map"  instead of "ConfigMap" , if this is the case, we need to fix the code and not the test.

Reply:
This is strange, cause actually I didn't do this change here